presto-main/src/test/java/com/facebook/presto/type/TestDecimalParametricType.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.type;
+
+import org.testng.annotations.Test;
+
+public class TestDecimalParametricType
+        extends AbstractTestDecimalFunctions
+{
+    @Test
+    public void decimalIsCreatedWithPrecisionAndScaleDefined()
+    {
+        assertDecimalFunction("CAST(1 AS DECIMAL(2, 0))", decimal("01"));
```

Comment:
I kinda followed @losipiuk instruction when creating this file.
They are kinda similar but cover other aspects, and this file test are not focused on conversion, rounding etc. It's just basic creation test to find out if one can create defined DECIMAL type variable as expected.
Other types don't spread they tests around so many files, but aren't that complex on the other side.

Anyway - I don't want to mess up things while implementing small and simple feature.